test/cram/test.t

```diff
@@ -0,0 +1,20 @@
+This is a cram test for the new executable.
+
+# Run the executable
+  $ dummy_terminal example.diff n n
```

Comment:
You are not updating the renderer with the new content, so the `keyboard_area`'s are never updated. However, e.g. with the help panel, the set of keys change!

If you try `dummy_terminal example.diff h q` you will see the end result is different from if you do that in the real application.